examples/blueprint.yaml

```diff
@@ -51,6 +86,9 @@ node_templates:
             use_external_resource: true
             resource_id: CFY-Internal
             vcloud_config: { get_property: [vcloud_configuration, vcloud_config] }
+        relationships:
+            - target: example_public_nat_for_net
+              type: cloudify.vcloud.public_nat_connected_to_net
```

Comment:
Same here - to be consistent with the direction of the relationship it make sense to name it 'net_connected_to_public_nat'